containers/concent-api/install-middleman-protocol.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash -e
+
+virtualenv_dir=/srv/http/virtualenv
+middleman_protocol_dir=/srv/http/middleman_protocol/
+source "$virtualenv_dir/bin/activate"
+
+cd "$middleman_protocol_dir"
+python setup.py install_lib
```

Comment:
Please add a comment that explains why you're using `install_lib` rather than `install`.